Code/Editor/TrackView/TrackViewDialog.cpp

```diff
@@ -2033,7 +2033,8 @@ void CTrackViewDialog::UpdateTracksToolBar()
                     continue;
                 }
 
-                name = pAnimNode->GetParamName(paramType);
+                AZStd::string paramName = pAnimNode->GetParamName(paramType);
+                name = paramName.c_str();
```

Comment:
I'm trying to understand why this fixes it.  Surely this would have been enough:
```
name = QString::fromUtf8(paramName.c_str());
```
since QString does in fact make a copy of the data.  This would also handle unicode in paramNames